### DISCUSSION TITLE
Is there a reason why different casing is used in the examples in the docs?

### DISCUSSION THREAD

**Discussion by pak11273** [nextauthjs/next-auth#3932] [Help]
### Question 💬

Different casing is used in the example schema [here](https://next-auth.js.org/adapters/models):
  
![image](https://user-images.githubusercontent.com/2309184/153726977-1a7000a3-db50-4d4e-aab0-cbe72dc84a69.png)

Is the casing important or can I just snake_case everything?  

### How to reproduce ☕️

see here: https://next-auth.js.org/adapters/models

### Contributing 🙌🏽

Yes, I am willing to help answer this question in a PR

**Comment by gregg-cbs** [comment]
I also cant understand this. Why are all properties underscored except for sessionToken, userId, providerAccount and emailVerified ? And from this kind of a package... must be a reason behind it?

And the title of this should be changed, this is how the package has been coded to work, its not just the documentation.
